src/pharmpy/statements.py

```diff
@@ -1776,6 +1776,29 @@ def full_expression(self, expression):
             expression = expression.subs({statement.symbol: statement.expression})
         return expression
 
+    def insert_before(self, statement, new_statement):
+        """Insert a statement just before another statement
+
+        Parameters
+        ----------
+        statement : Statement
```

Comment:
Since there is no Statement class, this should be `statement : Assignment or ODESystem`

Reply:
Ok, I'll add a Statement base class. This could also be used for `isinstance` checks.